test/test_screen.py

```diff
@@ -15,24 +15,26 @@ def setUpClass(self):
         set_root(__file__)
 
     def setUp(self):
-        surf.fill((0, 0, 0))
         self.screen = Screen(surf)
+        self.screen.clear()
```

Comment:
@lordmauve Is it fine to mix some small cleanup commits with fixes or new features?

Reply:
Yeah, I'm not that strict :)